Sources/CacheAdvance/CacheReader.swift

```diff
@@ -62,9 +62,6 @@ final class CacheReader {
         case .emptyRead:
             guard !previousReadWasEmpty else {
                 // If the previous read was also empty, then the file has been corrupted.
-                // Two empty reads in a row means that offsetInFileAtEndOfNewestMessage is incorrect.
-                // This inconsistency is likely due to a crash occurring during a message write, between truncating the file and updating the header values to reflect that the file was truncated.
-                // This file is actually empty, despite what the header tells us.
```

Comment:
We've determined that our attribution of how the file got corrupted is incorrect. Since we don't know how this happened, we are deleting the explanation.